src/components/Thread/Thread.vue

```diff
@@ -674,10 +674,13 @@ export default {
         text_color (message) {
             try {
                 let colorString;
-                if (message.message_from)
-                    colorString = this.getColor(message);
-                else // Otherwise default color
+
+                if (message.type == 1) { // Message was sent by the user, use default color
                     colorString = this.color;
+                }
+                else {
```

Comment:
[nit] the `else` clause should be formatted onto the previous line

Reply:
Actually - we can do without the block braces all together since it's a one-liner for the `if` and `else`